README.md

```diff
@@ -39,6 +39,7 @@ A curated list of awesome TON ecosystem.
 - [GitHub: copperbits/TON](https://github.com/copperbits/TON) - Telegram Open Network group of research enthusiasts. New community baked developers tools (Docker, Explorers, etc) releases will posted there.
 - [Telegram TON Research public group](https://t.me/ton_research) - With fan-developer updates about status of peoject and point to ask your quesitons.
 - [Telegram chat in Russian](https://t.me/TONgramDev)
+- [TON Forum Russia](https://tontalk.ru)
```

Comment:
2) To make it more organized: let's do it in this way:

I believe we need to create separate sections of the international community and sections for local chats/forums (like Russian, Chinese and so on). Let's move Russian chan and Forum there.